src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -296,16 +296,17 @@ internal static class WellKnownTypeNames
         public const string SystemThreadingInterlocked = "System.Threading.Interlocked";
         public const string SystemThreadingMonitor = "System.Threading.Monitor";
         public const string SystemThreadingSpinLock = "System.Threading.SpinLock";
-        public const string SystemThreadingTasksTask1 = "System.Threading.Tasks.Task`1";
-        public const string SystemThreadingTasksTask = "System.Threading.Tasks.Task";
+        public const string SystemThreadingTasksGenericTaskCompletionSource = "System.Threading.Tasks.TaskCompletionSource`1";
```

Comment:
I think you meant to get rid of these two.

Reply:
Or get rid of SystemThreadingTasksGenericValueTask, and rename SystemThreadingTasksGenericTaskCompletionSource  to SystemThreadingTasksTaskCompletionSource1.

Reply:
Thank you, I missed it when I did the conflict merge.